packages/next-app/src/components/ImageDialog.tsx

```diff
@@ -0,0 +1,29 @@
+"use client";
+
+import * as Dialog from "@radix-ui/react-dialog";
+import { ArrowTopRightIcon } from "@radix-ui/react-icons";
+import classNames from "classnames";
```

Comment:
minor thing but you imported as `cn` in other files

Reply:
https://github.com/fsubal/eslint-plugin-classnames/blob/main/docs/rules/prefer-classnames-function.md I think we could use this, wdyt @devjoaov ?